tests/integration/infrastructure/rpc/test_simulator_sessions.py

```
@@ -0,0 +1,440 @@
+async def test_delete_validator_uses_request_session(monkeypatch):
+    registry_instance = SimpleNamespace(delete_validator=AsyncMock())
+    validators_manager = SimpleNamespace(registry=registry_instance)
+
+    result = await endpoints.delete_validator(validators_manager, "0xabc")
+
+    assert result == "0xabc"
+    registry_instance.delete_validator.assert_awaited_once_with("0xabc")
+
+
+@pytest.mark.asyncio
+async def test_delete_all_validators_uses_request_session(monkeypatch):
```

Ruff (0.14.0)

Unused function argument: `monkeypatch`

(ARG001)